test-one/src/index-test.js

```diff
@@ -2,8 +2,8 @@ import { go } from "./index";
 import * as logger from "./helpers/logger";
 
 test("logged welcome message", async () => {
-  const expectedOutput =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! Please replace this message with your solution and feel free to add or modify other files as necessary. Good luck : )";
+  const expectedOutput = "Result-Above"
```

Comment:
What is this testing?

Reply:
I wasn't exactly sure how to test it if I'm honest. I suppose I could of hard coded the answer and tested against that